.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+name: Plugin CI
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+
+env:
+  PHP_CS_FIXER_IGNORE_ENV: 1
+  XDEBUG_MODE: coverage
+
+jobs:
+  tests:
+    name: "Tests ${{ matrix.php-version }} deps ${{ matrix.dependency-versions }}"
+    runs-on: ubuntu-22.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # normal, highest, non-dev installs
+        php-version: [ '8.2' ]
+        dependency-versions: [ 'highest' ]
+        include:
+          # testing lowest PHP version with the lowest dependencies
+  #        - php-version: '8.2'
```

Comment:
@Asisyas, are you going to uncommit it?

Reply:
Not now. Because in the old version of the framework there is no functionality to solve all the necessary tasks.